### PR TITLE
docs(zone.js): update zone.js bundle doc since the APF change

### PR DESCRIPTION
**Pull request by JiaLiPassion**
Since the PR #36540 change the zone.js bundles to Angular Package Format, the
bundle name/location are changed, so this PR updated the `README.md` doc for the
zone bundles.
Also add the recent added new bundles `zone-patch-message-port` doc.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

### CODE REVIEW

File: packages/zone.js/README.md [hunk 1]
```diff
@@ -73,17 +82,18 @@ And here are the additional optional patches not included in the main zone.js bu
 |webapis-rtc-peer-connection.js|patch for `RTCPeerConnection APIs`|
 |webapis-shadydom.js|patch for `Shady DOM APIs`|
 |zone-bluebird.js|patch for `Bluebird APIs`|
-|zone-error.js|patch for `Error Global Object`, supports remove `Zone StackTrace`|
+|zone-error.js|patch for `Error Global Object`, supports removing `Zone StackTrace`|
```
Comment (mhevery) [inline packages/zone.js/README.md:85]: is it removing Error support or adding? I would think it would add error support, so the text is confusing.

**Review by mhevery — CHANGES_REQUESTED** [review]

File: packages/zone.js/README.md [hunk 2]
```diff
@@ -54,17 +54,26 @@ For more details, please
...
 
 |Bundle|Summary|
 |---|---|
-|zone.js|the default bundle, contains the most used APIs such as `setTimeout/Promise/EventTarget...`, also this bundle supports all evergreen and legacy (IE/Legacy Firefox/Legacy Safari) Browsers|
-|zone-evergreen.js|the bundle for evergreen browsers, doesn't include the `patch` for `legacy` browsers such as `IE` or old versions of `Firefox/Safari`|
-|zone-legacy.js|the patch bundle for legacy browsers, only includes the `patch` for `legacy` browsers such as `IE` or old versions of `Firefox/Safari`. This bundle must be loaded after `zone-evergreen.js`, **`zone.js`=`zone-evergreen.js` + `zone-legacy.js`**|
-|zone-testing.js|the bundle for zone testing support, including `jasmine/mocha` support and `async/fakeAsync/sync` test utilities|
+|zone.js|the default bundle, contains the most used APIs such as `setTimeout/Promise/EventTarget...`, it also supports differential loading by importing  this bundle using `import zone.js`, that means in legacy browsers it includes some additional patches|
```
Comment (mhevery) [inline packages/zone.js/README.md]: what additional patches does it include in legacy?

**Review by mhevery — APPROVED** [review]
69c771a48e contains my fixes

File: packages/zone.js/README.md (hunk 1 above)
Comment (JiaLiPassion) [reply packages/zone.js/README.md]: Yeah, it is confusing, I have updated the text to
```
|`zone-error.js`|patch for `Error Global Object`, supports adding zone information to stack frame, and also removing unrelated stack frames from `zone.js` internally|
```

File: packages/zone.js/README.md (hunk 2 above)
Comment (JiaLiPassion) [reply packages/zone.js/README.md]: Now there are two patches in `legacy` bundle.
1. the old Web Components `registerElement` patch.
2. some old browsers not support `EventTarget` interface, so we have to patch all the APIs such as `HTMLElement` one by one.

**Pull request closed without merging**